### DISCUSSION TITLE
What is the best overall protocol?

### DISCUSSION THREAD

**Discussion by inventor7777** [pascallanger/DIY-Multiprotocol-TX-Module#981] [Polls]
Hey everyone! I’ve been using my Multi-module MT12 for almost 6 months now (I think I was one of the first in the US to get mine) 

Anyway, I want some feedback to make sure I’m not missing something.

**What protocol seems to have the most features, range, and telemetry support?**
I personally (after going through the supported protocols) think FlySky AFHDHS 2A is the best (it supports telemetry, remapping, fail safe, and it has excellent range). I mostly use the IA6B and the IA4B (which, for some reason, is pretty hard to find now) along with the FlySky telemetry sensors.

However, I haven’t kept up much with all the supported protocols and if something has changed, I want to know! 

So my real question is: **What protocol is your favorite and why?**

**Comment by geofrancis** [comment]
ask this in a forum or reddit, github is for software development.
